go/test/endtoend/vreplication/cluster.go

```diff
@@ -27,6 +27,7 @@ var (
 	mainClusterConfig     *ClusterConfig
 	externalClusterConfig *ClusterConfig
 	extraVTGateArgs       = []string{"-tablet_refresh_interval", "10ms"}
+	extraVtctldArgs       = []string{"-remote_operation_timeout", "600s", "-topo_etcd_lease_ttl", "120"}
```

Comment:
Not RPC related, the operation timeout gives us more time for VDiffs etc as we add new tests, tables get larger, CI is slow, etc. The etcd lease extension ensures that the keyspace lock does not timeout during the two TABLE LOCKs if a transaction is timed out in each one (30s trx timeout). I made them large simply to avoid flappy tests. 

I can remove them though if you want? They're not strictly needed here.

Reply:
If it helps keep the tests less flaky, should be fine to keep it.